prowler/lib/outputs/ocsf/ocsf.py

```diff
@@ -59,21 +58,24 @@ def transform(self, findings: List[Finding]) -> None:
                 finding_severity = getattr(
                     SeverityID, finding.severity.capitalize(), SeverityID.Unknown
                 )
-                finding_status = self.get_finding_status_id(
-                    finding.status, finding.muted
-                )
+                finding_status = self.get_finding_status_id(finding.muted)
 
                 detection_finding = DetectionFinding(
+                    message=finding.status_extended,
```

Comment:
Why this new field? I saw that it is recommended but it is the same as `status_detail`

Reply:
Yes, but I think it is good to see it at first.